### PR TITLE
docs: Fix problems with the 'rate-limit-service' page

### PR DESCRIPTION
**Pull request by LukeShu**
- Fix punctuation
- Proof-read for "Ambassador" vs "the Ambassador API Gateway" vs "the Ambassador Edge Stack"
- Clarify what's going on with `v1/rls.proto` vs `v2/rls.proto`

Fixes https://github.com/datawire/ambassador/issues/2828

### CODE REVIEW

**Comment by LukeShu** [comment]
Before merging, I'm waiting on the website preview to make sure that the code block is rendering in a sane way.

**Comment by LukeShu** [comment]
The trick is to use

    ```diff

instead of

    ```patch

It's doing something wrong with trimming leading whitespace from the first line, but that's not particularly important here.

File: docs/topics/running/services/rate-limit-service.md [hunk 1]
```diff
@@ -53,11 +60,51 @@ spec:
...
+> (`pb.lyft.ratelimit.RateLimitService`) to the one defined in
+> [`v2/rls.proto`][] (`envoy.service.ratelimit.v2.RateLimitService`):
+>
+> - In some future version of Ambassador, there will be a setting to control
+>   which name is used; with the default being the current name; it will be
+>   opt-in to the new name.
+>
+> - In some future version of Ambassador after that, *no sooner than Ambassador
+>   1.6.0*, the default value of that setting swill change; making it opt-out
+>   from the new name.
```
Comment (kflynn) [inline docs/topics/running/services/rate-limit-service.md:81]: `s/swill/will/` -- and I'd probably make this "no sooner than 1.7.0" at this point.

File: docs/topics/running/services/rate-limit-service.md [hunk 2]
```diff
@@ -53,11 +60,51 @@ spec:
...
+>
+> - In some future version of Ambassador after that, *no sooner than Ambassador
+>   1.7.0*, the setting will go away, and Ambassador will always use the new
```
Comment (kflynn) [inline docs/topics/running/services/rate-limit-service.md:84]: Probably 1.8.0.

File: docs/topics/running/services/rate-limit-service.md [hunk 3]
```diff
@@ -53,11 +60,51 @@ spec:
...
+>   name.
+>
+> In the mean-time, implementations of `RateLimitService` are encouraged to
```
Comment (kflynn) [inline docs/topics/running/services/rate-limit-service.md:87]: `s/mean-time/meantime/` since, y'know, it's not 100 years ago. :wink:

**Pull request merged**